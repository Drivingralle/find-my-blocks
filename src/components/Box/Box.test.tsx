import React from "react";
import renderer from "react-test-renderer";
import { Box } from "./";

it("render correctly", () => {
  const tree = renderer.create(<Box>Box</Box>).toJSON();
  expect(tree).toMatchSnapshot();
});

it("render correctly as a span", () => {
  const tree = renderer.create(<Box tag="span">Box</Box>).toJSON();
  if (tree) {
    expect(tree.type).toBe("span");
  }
});

it("render classNames correctly", () => {
  const tree = renderer.create(<Box className="raptors">Box</Box>).toJSON();
  if (tree) {
    expect(tree.props.className).toBe("raptors");
    expect(tree).toMatchSnapshot();
  }
});
