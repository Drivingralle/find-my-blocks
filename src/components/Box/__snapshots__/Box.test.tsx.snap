// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box component should render classNames correctly 1`] = `
<div
  className="raptors"
>
  Box
</div>
`;

exports[`Box component should render correctly 1`] = `
<div>
  Box
</div>
`;
